cfe_internal/update/update_policy.cf

```diff
@@ -236,6 +236,16 @@ bundle agent cfe_internal_update_policy_cpv
       action => u_immediate,
       classes => u_if_repaired("validated_updates_ready");
 
+    am_policy_hub::
+
+      "$(sys.workdir)$(const.dirsep)modules"
```

Comment:
I don't think that only the hub should do this. If we are moving to having MPF put modules in place, I think that should happen on each host.